code/formatters.py

```diff
@@ -148,6 +156,7 @@ def formatter_function(i, word, _):
     "allcaps": formatters_dict["ALL_CAPS"],
     "alldown": formatters_dict["ALL_LOWERCASE"],
     "camel": formatters_dict["PRIVATE_CAMEL_CASE"],
+    "no format": formatters_dict["NOOP"],
```

Comment:
Is there a reason why you added this in this PR? Seems like an unrelated change. Submit it in a different PR if you think that it's useful, best not to mix unrelated changes together unless they're really minor (like whitespace or comment formatting fixes).

Reply:
Yeah I explain it in the description. It's to undo formatting like strikethrough and later bold.

Reply:
Oops, shoulda read the description more carefully. Sorry!

Reply:
I'll remove it for now since maybe other formatters cover it but definitely related directly to this